WORKSPACE

```diff
@@ -34,6 +38,7 @@ local_repository(
 node_repositories(package_json = [
     "//examples/rollup:package.json",
     "@program_example//:package.json",
+    "//internal/e2e/rollup:package.json",
```

Comment:
I don't think we want this here - this package.json is governed by a `npm_install` rule so we should only ever access its contents in the external directory.

Reply:
I see, this is different from the one we `npm_install`